README.md

```diff
@@ -81,9 +81,10 @@ this:
 ## API
 
 MWoffliner provides also an API and therefore can be used as a NodeJS
-library. Here a stub example:
+library. Here a stub example that could go in your index.mjs file:
```

Comment:
mjs?

Reply:
Yes, rather than explaining that Node defaults to common JS style imports (`require ('foo')) and the various ways to get your Javascript file to do ESM imports, it seems simpler and more terse to just mention .mjs in passing.